src/regionnet.jl

```diff
@@ -34,8 +34,11 @@ else
     println("Trying to create a new region network...")
 
     # Load the network of counties
-    counties = readtable(joinpath(todata, "county-info$suffix.csv"), eltypes=[UTF8String, UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64, Float64, Float64])
-
+    if config["netset"] == "usa"
+    counties = readtable(joinpath(todata, "county-info.csv"), eltypes=[UTF8String, UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64, Float64, Float64])
+    else
+    counties = readtable(joinpath(todata, "county-info$suffix.csv"), eltypes=[UTF8String,     UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64,       Float64, Float64])
+    end
```

Comment:
It does work, but when we are building a model for only a state, we get a problem as the suffix is now the state fips and county-info-$suffix doesn't exist. This was a simple quick fix for that.